src/cls/rbd/cls_rbd_types.h

```diff
@@ -452,15 +650,15 @@ struct UserSnapshotNamespace {
   }
 };
 
-struct GroupSnapshotNamespace {
+struct GroupImageSnapshotNamespace {
```

Comment:
Nit: rename to ImageGroupSnapshotNamespace?

Reply:
GroupImageSnapshotNamespace makes it sound like it is the snapshot namespace type for a group Snapshot.
Renaming it to ImageGroupSnapshotNamespace makes it clearer that the namespace is for an Image snapshot.  Ideally the others should also be renamed to ImageUserSnapshotNamespace, ImageTrashSnapshotNamespace, ImageMirrorSnapshotNamespace etc.

Reply:
Sounds good. I'll make the changes.